pkg/apis/camel/v1/common_types.go

```diff
@@ -105,6 +105,8 @@ type MavenSpec struct {
 	// Deprecated: use IntegrationPlatform.Spec.Build.Timeout instead
 	Timeout      *metav1.Duration `json:"timeout,omitempty"`
 	Repositories []Repository     `json:"repositories,omitempty"`
+	// Maven build extensions
```

Comment:
Could we elaborate a bit the documentation, or that is straighforward?

Reply:
Hmm good question. The comment is enough in my opinion ? I've added a link to the Maven website https://maven.apache.org/guides/mini/guide-using-extensions.html to be safe

Reply:
Yes, a pointer to the Maven documentation is great 👍🏼.